tla-io/src/main/scala/at/forsyte/apalache/tla/imp/OpApplTranslator.scala

```diff
@@ -185,6 +194,13 @@ class OpApplTranslator(
           }
           OperEx(TlaOper.apply, NameEx(decl.name) +: args: _*)
 
+        case DeclUnit(decl: TlaConstDecl) =>
+          // call the user-defined operator
+          val args = node.getArgs.toList.map { p =>
+            exTran.translate(p)
+          }
+          OperEx(TlaOper.apply, NameEx(decl.name) +: args: _*)
```

Comment:
I don't know Scala very well; didn't know how to combine this and the above into a single `case`.

Reply:
Yes, change the above from 
```scala
case DeclUnit(decl: TlaOperDecl) =>
```
to 
```scala
case DeclUnit(decl: TlaOperDecl | TlaConstDecl) =>
```

Reply:
or possibly just drop the forced annotation, i.e.
```scala
case DeclUnit(decl) =>
```

Reply:
I don't think you can bind variables in pattern alternatives?

The second suggestion

> or possibly just drop the forced annotation

works, though.